services/state/token/account.proto

```diff
@@ -0,0 +1,176 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "basic_types.proto";
+import "crypto_approve_allowance.proto";
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.token">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * First-draft representation of a Hedera Token Service account entity in the network Merkle tree.
+ *
+ * As with all network entities, an account has a unique entity number, which is usually given along
+ * with the network's shard and realm in the form of a shard.realm.number id.
```

Comment:
Not a blocker for this PR, but we really should have better docs here. These schemas will generate Java objects that we will use in our code. So if I'm working on some code and I want to understand the Account class, I'm going to be looking at and reading that javadoc. So It would be nice to describe more what an account is, how it fits into our system, etc. At least at a high level.